runtime/common/src/main/kotlin/kotlinx/serialization/context/Context.kt

```diff
@@ -5,34 +5,50 @@ package kotlinx.serialization.context
 import kotlinx.serialization.*
 import kotlin.reflect.KClass
 
+/**
+ * Serial context is a runtime mechanism used by [ContextSerializer] and [PolymorphicSerializer]
+ * to obtain serializers which were not found at compile-time by the serialization plugin.
```

Comment:
>to obtain serializers which were not found at compile-time

This statement is misleading.
Not to obtain missing serializers, but to override ones or to provide them in the runtime.
It's not a fallback, it's opt-in mechanism